public/help/help.md

```diff
@@ -30,3 +31,91 @@ The following procedure describes adding and updating settings of custom tracks.
 4. If simplifying the BED file chunk or graph is possible, users will see a "Simplify Off" button, which when clicked with toggle to "Simplify On". This option enables vg simplify, which would remove small snarls. This option will only appear when there aren't any reads to be displayed. 
 5. Click Go to see the selected tracks render in the visualization area.
 ![Go Button](helpGuideImages/img9.png)
+
+##### How to make link-to-view URLs
+Users can compose URLs that link to a specific view, or a visualization of genomic data. The link describes which tracks and files will be displayed. 
+
+###### Query String Encoding
+The link has a [query string](https://en.wikipedia.org/wiki/Query_string#Structure) that consists of key-value pairs separated by ampersands, where the keys are not URL-econded and the values are URL-encoded. 
+
+Keys can use brackets to encode hierarchical structures such as arrays and objects. 
+   * For arrays, put a number in brackets to assign a new value to or access a value in that entry in the array.
+   * For objects, put the key's name in brackets to assign a new value to or access a value in that entry in the object. 
+
+The key's name should **not** be in quotes.
+
+###### Fields that can be included in the link URL:
+1. `tracks` 
+
+   Information about tracks. Tracks are objects consisting of `trackFile`, `trackType`, and `trackColorSettings`.
+
+   A track object in JSON format might look like this:
+   ```
+   {
+      "trackFile": "exampleData/internal/snp1kg-BRCA1.vg.xg",
+      "trackType": "graph",
+      "trackColorSettings": {
+         "mainPalette": "greys",
+         "auxPalette": "ygreys",
+         "colorReadsByMappingQuality": false
+      }
+   }
+   ```
+
+   The collection of tracks wil be encoded as query string parameters starting with tracks.
+
+   * The `trackFile` is the path (from the server working directory) or URL (any HTTP/HTTPS URL) to the track file. 
+     * Examples:
+         * File in server working directory: `tracks[0][trackFile]=exampleData/internal/snp1kg-BRCA1.vg.xg`.
+         * HTTP/HTTPS URL: `tracks[0][trackFile]=https://public.gi.ucsc.edu/~anovak/graphs/trivial-brca1.vg`.
+   * The `trackType` specifies the type of the track, which can be `graph`, `haplotype`, `read`.
+      * Example: `tracks[0][trackType]=graph`.
+   * `trackColorSettings` (optional): the color information for the track. It has subfields:
```

Comment:
These sibling bullets for `trackFile`, `trackType`, and `trackColorSettings` should all use the same sentence structure.